docs/GettingStartedDocs/GettingStarted.Windows.md

```diff
@@ -233,9 +233,10 @@ The following summary will assume that the contents were extracted to `C:\Intel
 4. Install the DCAP nuget packages:
     - The standalone `nuget.exe` [CLI tool](https://dist.nuget.org/win-x86-commandline/latest/nuget.exe) can be used to do this from the command prompt:
       ```cmd
-      nuget.exe install EnclaveCommonAPI -ExcludeVersion -Source "C:\Intel SGX DCAP for Windows v1.2.100.49925\nuget" -OutputDirectory C:\openenclave\prereqs\nuget
       nuget.exe install DCAP_Components -ExcludeVersion -Source "C:\Intel SGX DCAP for Windows v1.2.100.49925\nuget" -OutputDirectory C\openenclave\prereqs\nuget
+      nuget.exe install EnclaveCommonAPI -ExcludeVersion -Source "C:\Intel SGX DCAP for Windows v1.2.100.49925\nuget" -OutputDirectory C:\openenclave\prereqs\nuget
```

Comment:
Could we please default the nuget output directory and have an option for the user to overwrite it?  cmake of host/CMakeLists.txt will fail because it expects the SGX libraries to be under <depot root>\prereqs\nuget

Reply:
Thanks for the comment @jazzybluesea ! What do you mean in this case by "default"?  As in, what do you suggest needs to change?

I see what you mean by host/CMakeLists.txt expecting these nuget packages to be in  ${CMAKE_SOURCE_DIR}/prereqs/nuget/… , but I'm not sure what you mean by default here!

Reply:
This is not a script but a set of manual instructions. Earlier in this doc, it has been established that `C:\openenclave` is the example root folder that the GitHub project was cloned into, I'm not sure anything needs to change here. 

If this comment should be tagged to install-windows-prereqs.ps1 file instead, then I would defer that to #2097 which focuses on the parameterization of this.

Reply:
We sorted it out, adding some changes to address this! Thanks Sergio! :)

Reply:
(Also thanks Simon for your insight!)

Reply:
Thank you John!